templates/_templating_scripting.py

```diff
@@ -211,6 +212,10 @@ def create_issue(title: str, body: str) -> None:
         )
         if any(issue["title"] == title for issue in existing_issues):
             return
+        if author in BOT_AUTHORS:
+            # if the author is a bot, we don't want to assign the issue to the bot,
+            # so instead choose a human author from the latest commit
+            author = get_commit_authors(commits[-1])[0]
```

Comment:
This doesn't feel completely safe as we can't predict every circumstance.

How about storing the result of line 190 and getting authors from there?

https://github.com/SciTools/.github/blob/6cfb2f9ab870267f4968f1c6e048359cf84d0409/templates/_templating_scripting.py#L190

Reply:
https://github.com/SciTools/.github/pull/75/commits/44e4b6917eebd88a682423c24c88a0dc48c6c445